### PR TITLE
Vanilla Csat Loadouts

### PR DESCRIPTION
**Pull request by Lazejun**
## What type of PR is this.
1. [ ] Bug
2. [X] Enhancement

### What have you changed and why?
Information:
Added Calls in Templates for Loadouts.
Added Loadouts for Alits,Tanoa,Enoch.
Keept them similar to Nato Loadouts.

### Please specify which Issue this PR Resolves.
closes #XXXX

### Please verify the following and ensure all checks are completed.

1. [X] Have you loaded the Mission in Local Host?
2. [ ] Have you loaded the Mission in a Dedicated Server?

### Is further testing or are further changes required?
1. [X] No
2. [ ] Yes (Please provide further detail below.)

********************************************************
Notes:

### CODE REVIEW

File: A3-Antistasi/Templates/Vanilla_Inv_CSAT_Altis.sqf [hunk 1]
```diff
@@ -24,19 +24,19 @@ CSATAmmoBox = "O_supplyCrate_F";
 //   PVP LOADOUTS AND VEHICLES   ///
 ////////////////////////////////////
 //PvP Loadouts
-CSATPlayerLoadouts = [
+CSATlayerLoadouts = [
```
Comment (SonorousMeerkat) [inline A3-Antistasi/Templates/Vanilla_Inv_CSAT_Altis.sqf]: Why was this replaced. Just made a typo. "CSAT**layer**Loadouts" should be player.

File: A3-Antistasi/Templates/Vanilla_Inv_CSAT_Enoch.sqf [hunk 2]
```diff
@@ -22,19 +22,19 @@ CSATAmmoBox = "O_supplyCrate_F";
 //   PVP LOADOUTS AND VEHICLES   ///
 ////////////////////////////////////
 //PvP Loadouts
-CSATPlayerLoadouts = [
+CSATlayerLoadouts = [
```
Comment (SonorousMeerkat) [inline A3-Antistasi/Templates/Vanilla_Inv_CSAT_Enoch.sqf]: And again

File: A3-Antistasi/Templates/Vanilla_Inv_CSAT_Tanoa.sqf [hunk 3]
```diff
@@ -22,19 +22,19 @@ CSATAmmoBox = "O_supplyCrate_F";
 //   PVP LOADOUTS AND VEHICLES   ///
 ////////////////////////////////////
 //PvP Loadouts
-CSATPlayerLoadouts = [
+CSATlayerLoadouts = [
```
Comment (SonorousMeerkat) [inline A3-Antistasi/Templates/Vanilla_Inv_CSAT_Tanoa.sqf]: _And again_

**Comment by Bob-Murphy** [comment]
@Spoffy 
This is cleared for merging.

**Pull request merged**